### DISCUSSION TITLE
Purfview's Faster-Whisper not working

### DISCUSSION THREAD

**Discussion by patrickkw** [SubtitleEdit/subtitleedit#7387] [General]
I found Purfview's Faster-Whisper could only generate an empty srt in the portable version.(I do not check with the install version) And it always ask me to do download again (Download Purfview's Faster-Whisper(Update) even I did it every time when I came into the Whisper interface. Any help?

**Comment by niksedk** [comment]
Check the "whisper_log.txt" in the SE data folder (also available in context menu).
Note that the large model requires up to 14 GB of free memory and high-end cpu.
Read more here: https://www.nikse.dk/subtitleedit/help#audio_to_text_whisper

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/bcd2a30b-38a7-4b3f-9f38-1c6818456e2c)


The update thing should be working again in next update in a few days.... or you can try the beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.0/SubtitleEditBeta.zip

**Comment by patrickkw** [comment]
Thanks. it solved. I have 8g memory only but it can be done by option --compute_type float32.

**Comment by Purfview** [comment]
What is your GPU model and VRAM size?
